build-tools/src/main/kotlin/org/jetbrains/kotlin/KotlinNativeTest.kt

```diff
@@ -152,7 +152,7 @@ open class KonanGTest : KonanTest() {
     override fun run() {
         doBeforeRun?.execute(this)
         runProcess(
-                executor = project.executor::execute,
+                executor = {project.executor.execute(it)},
```

Comment:
What have happened here? Is it an issue with Kotlin compiler?

Reply:
seems so, NSME was throwing: '''java.lang.NoSuchMethodError: kotlin.jvm.internal.FunctionReferenceImpl.<init>(ILjava/lang/Class;Ljava/lang/String;Ljava/lang/String;I)V
'"

Reply:
I'm going to investigate this then.